install/resources/settings.xml

```diff
@@ -737,7 +737,7 @@ min=0]]></optionscode>
 		</setting>
 		<setting name="postmaxavatarsize">
 			<title>Maximum Avatar Dimensions in Posts</title>
-			<description><![CDATA[The maximum dimensions for avatars when being displayed in a post. If an avatar is too large, it will automatically be scaled down.]]></description>
+			<description><![CDATA[The maximum dimensions for avatars when being displayed in a post (width and height separated by 'x' or '|') . If an avatar is too large, it will automatically be scaled down.]]></description>
```

Comment:
I can't remember, does the overall settings file version need updating for these to be picked up during the upgrade process..? https://github.com/effone/mybb/blob/a9e2d0f2a7ba4c08e276cf8c2a40745e9856e396/install/resources/settings.xml#L2

Reply:
We did something similar in 1.8.16, @dvz may guide better

Reply:
Yeah, I think it's fine actually, was just making sure.